packages/functions/events/askBirthday.ts

```diff
@@ -35,6 +36,12 @@ export const handler = handleEvent(
         return;
       }
 
+      await saveBirthday({
+        birthday: null,
+        teamId: userInfo.team_id || "",
```

Comment:
I would also include this in the check above and not save if team_id is not present. (Even though I don't think its very likely)